src/index.html

```diff
@@ -9,6 +9,9 @@
   <!-- prevent render loop from initializing on load -->
   <script>
     window['SohoConfig'] = { renderLoop: { noAutoStart: true }};
+
+    // @todo remove this when the loading of locales can be delayed in Soho
+   // window['Soho'].Locale.culturesPath = `/ids-enterprise-ng-demo/assets/ids-enterprise/js/cultures/`;
```

Comment:
I mean this part @pwpatton with the delay on locale (is a comment but i think not needed)

Reply:
I see, got it

Reply:
```

    // @todo remove this when the loading of locales can be delayed in Soho
   // window['Soho'].Locale.culturesPath = `/ids-enterprise-ng-demo/assets/ids-enterprise/js/cultures/`;
```
Are you saying we cna remove these 2 lines?

Reply:
Im not 100% sure about that maybe @bthharper can answer. There is no longer a so delay so this comment `// @todo remove this when the loading of locales can be delayed in Soho` is definitely no longer a today. But i dont know if the culturesPath still needs to be set to this?

Reply:
Most likely probably should remove the comment and uncomment the line:
From
```
 // @todo remove this when the loading of locales can be delayed in Soho
   // window['Soho'].Locale.culturesPath = `/ids-enterprise-ng-demo/assets/ids-enterprise/js/cultures/`;
```

To
``` 
window['Soho'].Locale.culturesPath = `/ids-enterprise-ng-demo/assets/ids-enterprise/js/cultures/`;
```

Reply:
If we are not doing the loading of `en-US.js` now, I don't think the setting of `culturesPath` is required.  

It was just a suggestion of how we could configure the folder for `sohoxi.js`.